autoprecompiles/src/symbolic_machine_generator.rs

```diff
@@ -98,30 +98,23 @@ pub fn statements_to_symbolic_machine<A: Adapter>(
         let machine: SymbolicMachine<<A as Adapter>::PowdrField> =
             convert_machine(machine, &|x| A::from_field(x));
 
-        let instr = instr.clone().into_symbolic_instruction();
-
-        let instr = SymbolicInstructionStatement {
-            opcode: A::from_field(instr.opcode),
-            args: instr
-                .args
-                .iter()
-                .map(|a| A::from_field(a.clone()))
-                .collect(),
-        };
+        // It is sufficient to provide the initial PC, because the PC update should be
+        // deterministic within a basic block. Therefore, all future PCs can be derived
+        // by the solver.
```

Comment:
Because of this, the initial PC and all intermediate ones should be known at compile time. This should remove the column and all PC lookups!